src/sentry/workflow_engine/migrations/0085_crons_link_detectors_to_all_workflows.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 5.2.1
+from collections import defaultdict
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+def link_cron_detectors_to_all_workflows(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    """
+    For all detectors with type="monitor_check_in_failure" (cron monitors),
+    create DetectorWorkflow entries linking them to all workflows associated with
+    the same project (via AlertRuleWorkflow).
+
+    This mimics the previous behavior where cron issues would fire all alert rules
+    for a project.
+    """
+    Detector = apps.get_model("workflow_engine", "Detector")
+    DetectorWorkflow = apps.get_model("workflow_engine", "DetectorWorkflow")
+    Rule = apps.get_model("sentry", "Rule")
+    Workflow = apps.get_model("workflow_engine", "Workflow")
+
+    cron_detectors = Detector.objects.filter(type="monitor_check_in_failure")
```

Comment:
There are 100k rows in US here, so I'm hoping I can keep this simple and skip using the range wrapper